### PR TITLE
Upgrade decidim 0.17.0

### PR DESCRIPTION
**Pull request by aitorlb**
Upgrade decidim 0.17.0

### CODE REVIEW

**Comment by aitorlb** [comment]
@tramuntanal `decidim-sant_cugat` is deployed to heroku so it doesn't use the `whenever` gem.

**Comment by tramuntanal** [comment]
So they already have scheduled tasks via heroku addon?

**Comment by aitorlb** [comment]
@tramuntanal no, it will have to be configured in the next deploy.

File: Gemfile [hunk 1]
```diff
@@ -4,13 +4,8 @@ source 'https://rubygems.org'
 
 ruby '2.5.3'
 
-DECIDIM_VERSION = '~> 0.16.0'
+DECIDIM_VERSION = '~> 0.17.0'
```
Comment (tramuntanal) [inline Gemfile]: 0.17.1 is already out

Comment (aitorlb) [reply Gemfile]: When I try to update to `0.17.1` I get the following:
```console
$ bundle update decidim
Could not find gem 'decidim-dev (= 0.17.1)' in any of the gem sources listed in your Gemfile.
```

Comment (tramuntanal) [reply Gemfile]: @isaacmg410 has already notified @oriolgual about this

**Comment by isaacmg410** [comment]
Closing in favour of  https://github.com/AjuntamentdeSantCugat/decidim-sant_cugat/pull/47

**Pull request closed without merging**